CHANGELOG.md

```diff
@@ -3,21 +3,24 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.1.0dev
+## v1.1.0 - 22-11-2024 - Amaranth Filly
 
 ### `Added`
 
-- [#11](https://github.com/qbic-pipelines/vcftomaf/pull/11) - Add metromap
+- [#11](https://github.com/qbic-pipelines/vcftomaf/pull/11) - Add metromap (@FriederikeHanssen)
+- [#18](https://github.com/qbic-pipelines/vcftomaf/pull/18) - Add options to stop filtering by `PASS` (@famosab)
```

Comment:
```suggestion
- [#18](https://github.com/qbic-pipelines/vcftomaf/pull/18) - Add options to enable filtering by `PASS` via `--filter`, filtering was default in previous version (@famosab)
```
Hm I think its the other way around. The option allows to optionally switch on PASS filtering (whatever that is), in contrast the previous default was to have it switched on. So you changed the default, right? If yes, thats sort of noteworthy imho.

Reply:
yes you are right thats more descriptive